molgenis-data-mapper/src/main/java/org/molgenis/data/mapper/service/impl/MappingServiceImpl.java

```diff
@@ -183,14 +185,22 @@ public String applyMappings(MappingTarget mappingTarget, String entityName, bool
 		}
 
 		Repository<Entity> targetRepo;
-		if (!dataService.hasRepository(entityName))
+		if (!dataService.hasRepository(entityName) && !dataService.hasRepository("base_"+entityName))
 		{
+			targetMetaData.setPackage(defaultPackage);
 			// Create a new repository
 			targetRepo = runAsSystem(() -> dataService.getMeta().createRepository(targetMetaData));
 			permissionSystemService.giveUserEntityPermissions(getContext(), singletonList(targetRepo.getName()));
-		}
-		else
+		}else
 		{
+			targetMetaData.setPackage(null);
+			//if the entity is in the base package, this should be added to the name
+			if(dataService.hasRepository("base_"+entityName))
+			{
+				entityName = "base_" + entityName;
+				targetMetaData.setName(entityName);
+				targetMetaData.setLabel(entityName);
+			}
```

Comment:
if the entity was already created before in the base package, its entityname did not contain "base_", so I had to add it, but only if the entity was in base, otherwise it would put entities in another package in base as well.